src/core/main/worker/content_preparer.ts

```diff
@@ -270,6 +273,37 @@ export default class ContentPreparer {
       );
       manifestFetcher.start();
 
+      /**
+       * Set Representation.isCodecSupportedInWebWorker to true or false
+       * If the codec is supported in the current context.
+       * If MSE in worker is not available, the attribute is not set.
+       */
+      function updateCodecSupportInWorkerMode(manifestToUpdate: Manifest) {
```

Comment:
Can we move this function out of this scope as a top-level util?

This way both the (already hard-to-follow) method would be more readable, and that function's dependencies would be clearer (i.e. which part of the local state is depended on and which is not)